client-v2/src/keyboard/index.ts

```diff
@@ -63,7 +66,31 @@ export const createKeyboardActionMap = (store: Store): KeyboardBindingMap => ({
   'command+v': {
     title: 'Paste',
     description: 'Paste an entity',
-    action: bindActionMap(store, paste)
+    action: () =>
+      (store.dispatch as Dispatch)(async (dispatch: Dispatch) => {
```

Comment:
Me and Anna have just gone through this, and the double parenthesis are a little confusing -- we came up with something along the lines of 
```
action: async () => {
      const dispatch: Dispatch = store.dispatch;
      try {
        if (!navigator || !(navigator as any).clipboard) {
          throw new Error('No navigator available on paste');
        }
        // A temporary any here pending proper typings
        const content: string = await (navigator as any).clipboard.readText();
        if (!content) {
          return;
        }
        const contentData: RefDrop = { type: 'REF', data: content };
        const articleFragment = await dispatch(
          createArticleFragment(contentData)
        );
        if (!articleFragment) {
          return;
        }
        dispatch(
          insertClipboardArticleFragment('clipboard', 0, articleFragment.uuid)
        );
      } catch (e) {
        Raven.captureMessage(`Paste to clipboard failed: ${e.message}`);
      }
```
-- what do you think?

Reply:
Yes, this is much better! I will change it.